overlord/ifacestate/ifacemgr.go

```diff
@@ -137,6 +138,20 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 	return m, nil
 }
 
+// AppArmorPromptingRunning returns true if prompting is running.
```

Comment:
this needs a not that it can only be called after the StartUp of ifacestate manager